app/controllers/experiments_controller.rb

```diff
@@ -630,4 +633,22 @@ def sort_my_modules(records, sort)
       records
     end
   end
+
+  def generate_two_by_two_connections(src_tasks, dest_tasks)
+    return if src_tasks.empty? || dest_tasks.empty?
+
+    src_tasks.each do |src|
+      dest_tasks.each do |dest|
+        Connection.create!(input_id: dest.id, output_id: src.id)
+      end
+    end
+  end
+
+  def get_dest_my_modules(my_module)
```

Comment:
These methods are not needed (as we already have these defined on MyModule), but as a general rule, in Ruby we don't use `get_` prefixes, as they are redundant. In this case, a more proper name would just be `destination_my_modules`.